README.md

```diff
@@ -31,28 +29,136 @@ If you need BPMN examples, you can use resources from
 - [bpmn-visualization BPMN test diagrams](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/bpmn-files/README.md)
 
 
-## Examples
+## 🌏 Browser Support
 
-Want to know more about `bpmn-visualization` usage and extensibility? Have a look at the
-[__:fast_forward: live examples site__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/index.html).
+| <img src="https://user-images.githubusercontent.com/1215767/34348387-a2e64588-ea4d-11e7-8267-a43365103afe.png" alt="Chrome" width="16px" height="16px" /> Chrome | <img src="https://user-images.githubusercontent.com/1215767/34348383-9e7ed492-ea4d-11e7-910c-03b39d52f496.png" alt="Firefox" width="16px" height="16px" /> Firefox | <img src="https://user-images.githubusercontent.com/1215767/34348394-a981f892-ea4d-11e7-9156-d128d58386b9.png" alt="Safari" width="16px" height="16px" /> Safari | <img src="https://user-images.githubusercontent.com/1215767/34348380-93e77ae8-ea4d-11e7-8696-9a989ddbbbf5.png" alt="Edge" width="16px" height="16px" /> Edge | <img src="https://user-images.githubusercontent.com/1215767/34348590-250b3ca2-ea4f-11e7-9efb-da953359321f.png" alt="IE" width="16px" height="16px" /> Internet Explorer |
+| :---------: | :---------: | :---------: | :---------: | :---------: |
+| Yes | Yes | Yes | Yes | No |
 
-For more technical details and how-to, go to the [bpmn-visualization-examples](https://github.com/process-analytics/bpmn-visualization-examples/)
-repository.
 
+## 🎨 Features
+
+`bpmn-visualization` is in early development stages and is subject to changes prior to the `1.0.0` release.\
+\
+Implemented features:
+- Rendering of the most common BPMN elements.  
+Notice that there is currently no plan to support `Conversation` and `Choreography`.
+
+Current implementation:
+- The [packaging](https://github.com/process-analytics/bpmn-visualization-js/milestone/18) of the library.
+
+Future features:
+- BPMN extensions
+- The library extension points
+- Display options for execution data with interactive capacities.
 
-# Roadmap
 
-`bpmn-visualization` is in early development stages and is subject to changes prior to the `1.0.0` release.
+## 📋 Requirements
```

Comment:
:warning: these are build requirements, I don't understand why they are now here as we describe usage information

Reply:
I probably don't understand everything, but for me it's not only build requirements. If a user want install our library in his application, he need the same requirements ? @aibcmars ?

Reply:
in fact these are for build
when we will publish the package and someone will install it from npm he will be able to use it directly as a bundle

Reply:
ok, I'll move it ;)